### PR TITLE
Editor Settings Refinement

### PR DESCRIPTION
**Pull request by ypinbong**

### CODE REVIEW

File: src/components/ColorInput.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+interface Props {
+  value?: string;
+  onChange: Dispatch<SetStateAction<string>>;
+  changeHandler: () => void;
+}
+
+export const ColorInput = ({ value, onChange, changeHandler }: Props) => {
+  const classes = useStyles();
+
+  return (
```
Comment (sunnywx) [inline src/components/ColorInput.tsx:14]: use color picker

File: src/components/ColorInput.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    <div className={classes.colorPicker}>
+      <input
+        type='color'
+        value={value}
+        onChange={(e) => onChange(e.target.value)}
+        className={classes.colorInput}
+        />
+      <input
```
Comment (sunnywx) [inline src/components/ColorInput.tsx:22]: use mui's TextField

File: src/components/props-panel/index.tsx [hunk 3]
```diff
@@ -26,7 +31,20 @@ export default function PropsPanel({canvas}: Props) {
   const onDeleteSelected = () => {
     canvas?.deleteSelected()
   }
-  
+
+  const fontArr: string[] = [
+    "Arial",
+    "Times New Roman",
```
Comment (sunnywx) [inline src/components/props-panel/index.tsx:37]: Are these fonts windows/mac support natively ?

**Review by sunnywx — APPROVED** [review]

File: src/components/props-panel/index.tsx (hunk 3 above)
Comment (ypinbong) [reply src/components/props-panel/index.tsx:37]: Yeah, I asked chatgpt for the common ones haha

**Pull request merged**